projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -93,7 +93,11 @@ export class PageTimeRangePreferenceService {
   }
 
   public getGlobalDefaultTimeRange(): TimeRange {
-    return new RelativeTimeRange(new TimeDuration(1, TimeUnit.Day));
+    return this.maybeGetCurrentTimeRange() ?? new RelativeTimeRange(new TimeDuration(1, TimeUnit.Day));
```

Comment:
This seems off to me - it would make the default _always_ the current after startup. Maybe instead, have the `initializeTimeRange` subscription save the requested time range and expose that?

Reply:
I only want to call this when FF is disabled. I missed that it is used elsewhere. I can just update this method to have the new behavior. What do you think?

```
// When FF is disabled
return () => this.getGlobalDefaultTimeRange();
```

Reply:
Updated. @aaron-steinfeld  please check